### ISSUE TITLE
Model training summary message

### ISSUE THREAD

**Issue by leihao1** [leihao1/CNN-CIFAR-10-Image-Classification#2]
should print validation set acc instead of test set acc
